packages/flutter/lib/src/services/message_codecs.dart

```diff
@@ -179,9 +179,7 @@ class JSONMethodCodec implements MethodCodec {
 ///  * null: null
 ///  * [bool]\: `java.lang.Boolean`
 ///  * [int]\: `java.lang.Integer` for values that are representable using 32-bit
-///    two's complement; otherwise, `java.lang.Long` for values that are
-///    representable using 64-bit two's complement; otherwise,
-///    `java.math.BigInteger`.
+///    two's complement; otherwise, `java.lang.Long`
```

Comment:
nit - missing period

Reply:
Thanks. Most of the other bullets don't use periods, so I've removed them consistently.